### ISSUE TITLE
`print` is causing internal errors

### ISSUE THREAD

**Issue by shriram** [brownplt/pyret-lang#1762]
This program:
```
test-table = table: col-1, col-2, col-3
  row: 1, 2, 3
  row: 2, 3, 4
  row: 3, 4, 5
end
print(test-table.row-n(0))
```
gives the error "Internal errors prevented this error message from being shown. Please report this as a bug."

Similarly this program:
```
import gdrive-sheets as GS

titanic-raw-loader = GS.load-spreadsheet("1ZqZWMY_p8rvv44_z7MaKJxLUI82oaOSkClwW057lr3Q")

titanic-raw = load-table: survived :: Number, pclass :: Number, raw-name :: String, sex :: String, age :: Number, sib-sp :: Number, par-chil :: Number, fare :: Number source: titanic-raw-loader.sheet-by-name("titanic", true) end

print(titanic-raw.row-n(0))
```

User is on Chrome version: 128.0.6613.138, which is the latest version as of this writing. I am unable to reproduce this, but I'm on macOS, whereas user is on Windows 11, Version: 23H2.

Bug reported by Sophia Wu.

**Comment by blerner** [comment]
Does not reproduce on Win10 on Firefox 130 or on Chrome 128.  I'd be very surprised if Win11 + Chrome is somehow different here from Win10 + Chrome...

**Comment by shriram** [comment]
I agree this is very surprising. Any ideas what I can suggest to the
student to try that would help us?

**Comment by blerner** [comment]
Not a clue -- it doesn't reproduce on Linux, either.  I'm tempted to say the student needs to clear their browser cache and try again; it sounds too spurious and idiosyncratic to track down, so far...
